docs/openid.md

```diff
@@ -549,6 +549,46 @@ oidc_providers:
         display_name_template: "{{ user.preferred_username|capitalize }}"
 ```
 
+### Shibboleth with OIDC Plugin
+
+[Shibboleth](https://www.shibboleth.net/) is an open Standard IdP solution widely used by Universities.
+
+1. Shibboleth needs the [OIDC Plugin](https://shibboleth.atlassian.net/wiki/spaces/IDPPLUGINS/pages/1376878976/OIDC+OP) installed and working correctly.
+2. Create a new config for the new endpoint on IDP Side:
+```yaml
+{
+        "client_id": "SOME-CLIENT-ID",
+        "client_secret": "SOME-SUPER-SECRET-SECRET",
+        "response_types": ["code"],
+        "grant_types": ["authorization_code"],
+        "scope": "openid profile email",
+        "redirect_uris": ["https://[synapse public baseurl]/_synapse/client/oidc/callback"]
+}
+```
+
+Synapse config:
+```yaml
+oidc_providers:
+  # Shibboleth IDP
+  #
+  - idp_id: shibboleth
+    idp_name: "Shibboleth Login"
+    discover: true
+    issuer: "https://YOUR-IDP-URL.TLD"
+    client_id: "YOUR_CLIENT_ID" #Note: not an URL because of Secial Chars
```

Comment:
What does the comment here mean? "Not an URL"? Why would this be a URL?

Reply:
you maybe could also use a URL Like "https://myawesomeservice.org" as Client_ID but that seems to make problems here.

Reply:
This should be something random, it looks like you have to fill this in manually as part of the OIDC plug-in?

Reply:
the client_id can be anything random, yes. But it must be the same on both sides: provider/client

Reply:
i did pull out the comment and did my hint below the code section.